### PR TITLE
- adds change notifications payload types in v1.0

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: api-reference/v1.0/resources/changenotification.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Methods
+
+None.
+
+## Properties
+
+| Property | Type | Description |
+|:---------|:-----|:------------|
+| changeType | string | Indicates the type of change that will raised the change notification. The supported values are: `created`, `updated`, `deleted`. Required. |
```
Comment (davidmu1) [inline api-reference/v1.0/resources/changenotification.md:26]: Indicates the type of change that will raise the change notification.

File: api-reference/v1.0/resources/changenotification.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+| clientState | string | Value of the **clientState** property sent specified in the subscription request (if any). The maximum length is 255 characters. The client can check that the change notification came from the service by comparing the value of the **clientState** property sent with the subscription with the value of the **clientState** property received with each change notification. Optional. |
```
Comment (davidmu1) [inline api-reference/v1.0/resources/changenotification.md:27]: Value of the **clientState** property sent in the subscription request (if any). The maximum length is 255 characters. The client can check whether the change notification came from the service by comparing the values of the **clientState** property. The value of the **clientState** property sent with the subscription is compared with the value of the **clientState** property received with each change notification.

File: api-reference/v1.0/resources/changenotification.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+| id | string | Unique ID for the notification. Optional. |
+| resource | string | The URI of the resource that emitted the change notification relative to `https://graph.microsoft.com`. Required. |
+| resourceData | [microsoft.graph.resourceData](resourcedata.md) | The content of this property depends on the type of resource being subscribed to. Required. |
```
Comment (davidmu1) [inline api-reference/v1.0/resources/changenotification.md:30]: Is the user going to know where to find these types?

File: api-reference/v1.0/resources/changenotification.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+| sequenceNumber | int | A number in sequence for a notification, to help the client app identify if notifications are in sequence or if a notification is missing. Optional. |
+| subscriptionExpirationDateTime | [dateTime](https://tools.ietf.org/html/rfc3339) | The expiration time for the subscription. Required. |
+| subscriptionId | string | The unique identifier of the subscription that generated the notification. |
+| tenantId | guid | The unique identified of the tenant from which the change notification originated. |
```
Comment (davidmu1) [inline api-reference/v1.0/resources/changenotification.md:34]: The unique identifier of the tenant from which the change notification originated.

File: api-reference/v1.0/resources/changenotification.md (hunk 1 above)
Comment (baywet) [reply api-reference/v1.0/resources/changenotification.md:26]:
```suggestion
| changeType | string | Indicates the type of change that will raise the change notification. The supported values are: `created`, `updated`, `deleted`. Required. |
```

File: api-reference/v1.0/resources/changenotification.md (hunk 3 above)
Comment (baywet) [reply api-reference/v1.0/resources/changenotification.md:30]: yes this really depends on what value they put in the `resource` property when creating the subscription.

File: api-reference/v1.0/resources/changenotification.md (hunk 4 above)
Comment (baywet) [reply api-reference/v1.0/resources/changenotification.md:34]:
```suggestion
| tenantId | guid | The unique identifier of the tenant from which the change notification originated. |
```

File: api-reference/v1.0/resources/changenotification.md (hunk 2 above)
Comment (baywet) [reply api-reference/v1.0/resources/changenotification.md:27]:
```suggestion
| clientState | string | Value of the **clientState** property sent in the subscription request (if any). The maximum length is 255 characters. The client can check whether the change notification came from the service by comparing the values of the **clientState** property. The value of the **clientState** property sent with the subscription is compared with the value of the **clientState** property received with each change notification. Optional. |
```

**Pull request merged**